File: Bronze/Bronze_4/C_files/Code/Memorizing_Triangles_10101.c
#include <stdio.h>

int main()
{
    int a, b, c;

    scanf("%d\n%d\n%d", &a, &b, &c);

    if (a + b + c == 180) {
        if (a == b && b == c && c == a)
            printf("Equilateral");
        else if (a == b || b == c || c == a)
            printf("Isosceles");
        else
            printf("Scalene");
    }
    else
        printf("Error");
    
    return 0;
}

File: Bronze/Bronze_4/Java_files/Code/Memorizing_Triangles_10101.java
import java.util.Scanner;

public class Memorizing_Triangles_10101 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int a = sc.nextInt();
        int b = sc.nextInt();
        int c = sc.nextInt();
        if (a + b + c == 180) {
            if (a == b && b == c && c == a)
                System.out.println("Equilateral");
            else if (a == b || b == c || c == a)
                System.out.println("Isosceles");
            else
                System.out.println("Scalene");
        }
        else
            System.out.println("Error");
    }
}
